HPXMLtoOpenStudio/resources/data/default_panels.csv

```diff
@@ -0,0 +1,29 @@
+Load Name,Voltage,Power Rating,Breaker Spaces,Units,Load Type,Notes
```

Comment:
Thanks for expanding this to include load type and breaker spaces. I was on the fence, but I think it makes sense to call them out both for clarity and so that they can be used directly